kw

```diff
@@ -22,9 +22,10 @@ KW_ETC_DIR="${XDG_CONFIG_HOME:-"${HOME}/.config"}/${KWORKFLOW}"
 # Set system-wide path
 if [[ -f "$KW_SYSTEM_WIDE_INSTALLATION" ]]; then
   KW_LIB_DIR='/usr/share/kw'
+  KW_DB_DIR="/usr/share/kw/database"
```

Comment:
We don't want kw database in a protected directory; it is better to be at the user's home.

Reply:
Ok.